packages/bsky/src/subscription/repo.ts

```diff
@@ -118,17 +118,16 @@ export class RepoSubscription {
       await this.handleCommit(msg)
     } else if (message.isHandle(msg)) {
       await this.handleUpdateHandle(msg)
-    } else if (message.isMigrate(msg)) {
-      await this.handleMigrate(msg)
     } else if (message.isTombstone(msg)) {
       await this.handleTombstone(msg)
+    } else if (message.isMigrate(msg)) {
+      // Ignore migrations
```

Comment:
yeah tbh im not sure appview will ever need to handle migrations. more of a bgs thing